src/test/resources/features/DeleteTournament.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Tournament
+  In order to use the app
+  As a tournament master
+  I want to delete a tournament
+
+  Scenario: Tournament Master deletes tournament
+    Given I login as "demoTM" with password "password"
+    And  I create a new tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    When I delete a tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    Then The response code is 204
+    And It does not exist a tournament with name "FirstTournament"
+
+  Scenario: Player tries to delete a tournament
+    Given I login as "demoTM" with password "password"
+    And  I create a new tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    Given I login as "demoP" with password "password"
+    When I delete a tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    Then The response code is 403
+
+  Scenario: Remove a tournament that does not exist
+    Given I login as "demoTM" with password "password"
+    When I delete a tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    Then The response code is 404
+
+    """
```

Comment:
This is recognized by Cucumber as a String parameter of the last step. The appropiate way to comment lines in a .feature file is with a # at the line start. This is causing a Travis failure.